test/uk/gov/hmrc/plasticpackagingtax/returns/controllers/home/SignOutControllerSpec.scala

```diff
@@ -0,0 +1,175 @@
+/*
```

Comment:
I’m assuming the timeout dialog can be dismissed before the timeout limit is reached and the user remains authenticated. Can we/do we need to test around this here?

Reply:
Hard to test this one. What we normally do is show case it locally